cmd/influx/task.go

```diff
@@ -53,3 +55,312 @@ func newTaskCreateCmd() *cli.Command {
 		},
 	}
 }
+
+func newTaskFindCmd() *cli.Command {
+	var params task.FindParams
+	params.Limit = TaskMaxPageSize
+	flags := append(commonFlags(), getOrgFlags(&params.OrgParams)...)
+	flags = append(flags, []cli.Flag{
+		&cli.StringFlag{
+			Name:        "id",
+			Usage:       "task ID",
+			Aliases:     []string{"i"},
+			Destination: &params.TaskID,
+		},
+		&cli.StringFlag{
+			Name:        "user-id",
+			Usage:       "task owner ID",
+			Aliases:     []string{"n"},
+			Destination: &params.UserID,
+		},
+		&cli.IntFlag{
+			Name:        "limit",
+			Usage:       "the number of tasks to find",
+			Destination: &params.Limit,
+		},
+	}...)
+	return &cli.Command{
+		Name:    "list",
+		Usage:   "List tasks",
+		Aliases: []string{"find", "ls"},
+		Flags:   flags,
+		Before:  middleware.WithBeforeFns(withCli(), withApi(true)),
+		Action: func(ctx *cli.Context) error {
+			api := getAPI(ctx)
+			client := task.Client{
+				CLI:      getCLI(ctx),
+				TasksApi: api.TasksApi,
+			}
+			return client.Find(ctx.Context, &params)
+		},
+	}
+}
+
+func newTaskRetryFailedCmd() *cli.Command {
+	var params task.RetryFailedParams
+	params.RunFilter.Limit = 100
+	params.TaskLimit = 100
```

Comment:
Same comment for these 2 lines, I think `Value` will have the same affect + nicer CLI help text